config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/EntityLoader.java

```diff
@@ -9,8 +9,11 @@
 import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
 
 import java.io.File;
+import java.util.Map;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Map'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=154&issues=AWekvNyWh8YERCevmIXA&open=AWekvNyWh8YERCevmIXA)

[ISSUE_KEY]: AWekvNyWh8YERCevmIXA